charts/portworx/templates/portworx-storageclasses.yaml

```diff
@@ -1,56 +1,9 @@
-kind: StorageClass
```

Comment:
was this intentional? This removes the default storage classes that the chart installs.

Reply:
No, that was meant to stay withing my fork of the repo. I'll update the PR to remove that.